### PR TITLE
chore: stop tracking timeline svg

### LINKED ISSUE #1091: Improve handling of timeline svg
**Please describe the feature**

As a developer, I want all files to behave well with respect to the version control so that I don't have to be confused or  spend time fixing it.

The file `frontend/src/assets/gemRepository/integratedModelsTimeline.svg` sometimes appears as manually updated when you move between branches and has to be restored/deleted/added.

**Acceptance Criteria**

- [ ] move the `integratedModelsTimeline.svg` out of the current repository
- [ ] create a new issue for creating the procedure to manage this file in the `data-files` repository

**Additional context**

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR closes #1091 .

**Type of change**  

- [x] Other
 
**List of changes made**  

- Removed timeline SVG.
- Stop tracking (add to `.gitignore`) timeline SVG.

**Testing**  

- A rebuild is needed due to new dependencies
- Instructions on how to test
1. Rebuild the project: `build-stack`.
2. Verify that `integratedModelsTimeline.svg` is not longer tracked in git: `git diff`.

**Further comments**  

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by inghylt — APPROVED** [review]
Nice work!

File: .gitignore [hunk 1]
```diff
@@ -96,3 +96,6 @@ pg/postgres-data
 
 # rollup visualizer output
 frontend/stats.html
+
+# timeline SVG
+frontend/public/assets/gemRepository/integratedModelsTimeline.svg
```
Comment (mihai-sysbio) [inline .gitignore:101]: I would opt for a more robust, filename-agnostic ignore rule:

```suggestion
frontend/public/assets/gemRepository/*.svg
```

**Review by mihai-sysbio — APPROVED** [review]
Looks good to me! I've suggested an idea below.

File: .gitignore (hunk 1 above)
Comment (inghylt) [reply .gitignore:101]: I don't see the need of this since we don't know what we might want to track or not in the future

Comment (e0) [reply .gitignore:101]: I don't have too strong opinion for this specific case. In general I agree with @inghylt in the sense that I don't want to anticipate too much about what might or might not happen in the future. This is not quite [premature optimization](https://stackify.com/premature-optimization-evil/) but the approach is similar.

**Pull request merged**